diff --git a/CMakeLists.txt b/CMakeLists.txt
index 7c8946f..292ce7f 100644
--- a/source_subfolder/CMakeLists.txt
+++ b/source_subfolder/CMakeLists.txt
@@ -185,6 +185,9 @@ set(CONAN_SETTINGS "build_type=${CMAKE_BUILD_TYPE};compiler.libcxx=${LIBCXX};com
 
 message(STATUS "CONAN_SETTINGS: ${CONAN_SETTINGS}")
 
+if(CONAN_EXPORTED)
+    conan_basic_setup(TARGETS)
+else()
 conan_cmake_run(REQUIRES
     catch2/2.13.6
     benchmark/1.8.0
@@ -193,6 +196,7 @@ conan_cmake_run(REQUIRES
     ENV CC=${CMAKE_C_COMPILER} CXX=${CMAKE_CXX_COMPILER}
     BUILD missing
 )
+endif()
 
 # configuration
 #############################################################################################
