diff --git a/CMakeLists.txt b/CMakeLists.txt
index 0806641..e3cfd31 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -191,6 +191,9 @@ endif ()

 message(STATUS "CONAN_SETTINGS: ${CONAN_SETTINGS}")

+if(CONAN_EXPORTED)
+    conan_basic_setup(TARGETS)
+else()
 conan_cmake_run(REQUIRES
     catch2/2.13.6
     benchmark/1.8.0
@@ -199,7 +202,7 @@ conan_cmake_run(REQUIRES
     ENV CC=${CMAKE_C_COMPILER} CXX=${CMAKE_CXX_COMPILER} LDFLAGS=${LINKFLAGS}
     BUILD missing
 )
-
+endif()
 # configuration
 #############################################################################################
 #conan
